drivers/tmp117: TMP117 driver (#22310)
RIOT-OS/RIOT: tests/drivers/tmp117/README.md

> # Test Application for tmp117 sensor

Reviewer: ```suggestion
## Test Application for TMP117 Sensor
```

This is a basic test for TMP117 driver.

> # Usage

Reviewer: ```suggestion
## Usage
```

This test initialize the sensor (address: 0x90) withe the following paramters:
	

> 	* conversion mode : continuous

Reviewer 2: ```suggestion
* conversion mode : continuous
```

* conversion cycle : 1s

> * averaging : 8 samples

Reviewer: ```suggestion
* conversion mode: continuous
* conversion cycle: 1s
* averaging: 8 samples
```

> After the initialization, the temperature value is displayed every 1s5.

Reviewer: ```suggestion
After the initialization, the temperature value is displayed every 1.5 seconds.
```
